src/gui/folder.cpp

```diff
@@ -1332,6 +1332,60 @@ void Folder::slotAboutToRemoveAllFiles(SyncFileItem::Direction dir, std::functio
     msgBox->open();
 }
 
+void Folder::removeLocalE2eFiles()
```

Comment:
@claucambra We should use selective sync blacklist and rely on a sync engine to remove files that we can't use.